packages/landing/src/attribution.ts

```diff
@@ -119,7 +119,12 @@ export class MapAttribution {
     this.zoom += 1;
 
     const bbox = this.mapboxBoundToBbox(this.bounds, Config.map.tileMatrix);
-    const filtered = attr.filter(bbox, this.zoom, Config.map.dateRange.dateAfter, Config.map.dateRange.dateBefore);
+    const filtered = attr.filter(
+      bbox,
+      this.zoom,
+      Config.map.filter.dateRange.dateAfter,
```

Comment:
could simplify to `date.before` `date.after` Im not sure you need to duplicate the `date` in both parts.

Reply:
Good point, just fixed. Thank you.